.github/workflows/tests.yml

```diff
@@ -13,37 +13,50 @@ jobs:
       fail-fast: true
       matrix:
         php: [7.3, 7.4, "8.0", 8.1, 8.2, 8.3]
-        laravel: [8, 9, 10, 11]
+        laravel: [8, 9, 10, 11, 12]
         stability: ["prefer-lowest", "prefer-stable"]
         include:
           - php: 8.4
             laravel: 11
             stability: "prefer-stable"
+          - php: 8.4
+            laravel: 12
+            stability: "prefer-stable"
         exclude:
+          - php: 7.3
+            laravel: 8 # Failed security advisory
```

Comment:
Failed security advisory due to unable to match any fixes https://github.com/advisories/GHSA-c2pc-g5qf-rfrf on PHP 7.3